### PR TITLE
create register splash

### LINKED ISSUE #9: Create Register Splash
image

- components/register/splash
- use the button component (someone else will work on this component)

### PR DESCRIPTION
**Pull request by twang0323**
image

### CODE REVIEW

File: src/components/register/Splash.jsx [hunk 1]
```diff
@@ -1,5 +1,20 @@
+// Splash.jsx
```
Comment (shahdivyank) [inline src/components/register/Splash.jsx]: remove comment

File: src/components/register/Splash.jsx [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+import React from "react";
+
 const Splash = () => {
-  return <div>Splash</div>;
+  return (
+    <div className="splash-container bg-code-black text-code-white min-h-screen flex flex-col items-center justify-center">
+      <h1 className="splash-title text-4xl mb-4 text-code-white">
```
Comment (shahdivyank) [inline src/components/register/Splash.jsx]: dont use h1 tag

File: src/components/register/Splash.jsx [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+        Congratulations!
+      </h1>
+      <h2 className="splash-welcome text-2xl mb-8">
```
Comment (shahdivyank) [inline src/components/register/Splash.jsx]: dont use h2 tag

File: src/components/register/Splash.jsx [hunk 4]
```diff
@@ -1,5 +1,20 @@
...
+        Welcome to <span className="text-code-pink">Apollo</span>!
+      </h2>
+      <button className="splash-button py-3 px-4 bg-code-purple text-code-white rounded-full">
```
Comment (shahdivyank) [inline src/components/register/Splash.jsx]: change this to a link, that takes u to dashboard

**Comment by twang0323** [comment]
I added the link, but when there is an underline. I tried using <a></a>. There is an error: Unhandled Runtime Error
Error: Invalid <Link> with <a> child. Please remove <a> or use <Link legacyBehavior>.
Learn more: https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor

File: src/components/register/Splash.jsx [hunk 5]
```diff
@@ -1,5 +1,20 @@
...
-  return <div>Splash</div>;
+  return (
+    <div className="bg-code-black text-code-white min-h-screen flex flex-col items-center justify-center">
+      <p className="text-4xl mb-4 text-code-white">Congratulations!</p>
+      <p className="text-2xl mb-8">
+        Welcome to <span className="text-code-pink">Apollo</span>!
+      </p>
+      <Link
+        href="/dashboard"
+        className="py-3 px-4 bg-code-purple text-code-white rounded-full"
```
Comment (shahdivyank) [inline src/components/register/Splash.jsx]: no-underline to remove the underline

Comment (twang0323) [reply src/components/register/Splash.jsx:12]: I added <a></a> for it to not show the underline for hyperlink but it runs an error, my previous comment wasn't complete. I pushed the code so you could see.

**Comment by twang0323** [comment]
image

File: src/components/register/Splash.jsx [hunk 6]
```diff
@@ -1,5 +1,19 @@
...
 const Splash = () => {
-  return <div>Splash</div>;
+  return (
+    <div className="bg-code-black text-code-white min-h-screen flex flex-col items-center justify-center">
+      <p className="text-4xl mb-4 text-code-white">Congratulations!</p>
+      <p className="text-2xl mb-8">
+        Welcome to <span className="text-code-pink">Apollo</span>!
+      </p>
+      <Link href="/dashboard">
+        <button className="py-3 px-4 bg-code-purple text-code-white rounded-full">
```
Comment (shahdivyank) [inline src/components/register/Splash.jsx]: remove the button, you can add the classnames to the link component and add a no-underline class to the link tag

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**